bin/makefile-linux.x86_64-x

```diff
@@ -25,7 +25,8 @@ DISPOPTFLAGS = -O2 -g3
 FPFLAGS =
 DFLAGS = -DFSERROR -DNEW_STORAGE -DLINUX -DAIX -DOLD_CURSOR -DUSETERMIOS \
 	-DBYTESWAP -DFORKCOMM -DNOFORN -DLOGINT  $(XFLAGS) \
-	-DRELEASE=351 -D__USE_BSD -DSYSVSIGNALS -DSYSVONLY
+	-DRELEASE=351 -D__USE_BSD -DSYSVSIGNALS -DSYSVONLY \
+	-D_GNU_SOURCE
```

Comment:
What GNU_SOURCE feature(s) are we depending on?

Reply:
```
../src/timer.c:689:55: error: use of undeclared identifier 'F_SETSIG'
  if (fcntl(ConnectionNumber(currentdsp->display_id), F_SETSIG, 0) < 0)
                                                      ^
```

It spits out this error building in a fresh Ubuntu VM without GNU_SOURCE, but no error with.

Reply:
OK, sounds good.  What's the right process on github to amend the commit message [Use of F_SETSIG requires _GNU_SOURCE; linux/termio.h became termios.h; etc.] (and possibly squash commits) as we merge this?

Reply:
I think GitHub lets you do a squash as you merge - can't see it on my end in this repo. I would say it's best for the person running the merge to do it so that we maintain the original commit history in the archived pull request.